python/afdko/buildcff2vf.py

```diff
@@ -297,6 +291,10 @@ def getCharStrings(self, num_masters, private=None, globalSubrs=None):
             charString = T2CharString(
                 program=program, private=private, globalSubrs=globalSubrs)
             t2List.append(charString)
+        # if default_idx is not 0, we need to move it to the right index.
+        if default_idx:
+            default_font_cs = t2List.pop(0)
+            t2List.insert(default_idx, default_font_cs) 
         return t2List
 
 
```

Comment:
One more thing: `glyphOrder` is not used in `_get_cs()` starting at line 301 below: